web/src/pages/Route/components/Step1/ProxyRewrite.tsx

```diff
@@ -100,14 +100,6 @@ const ProxyRewrite: React.FC<RouteModule.Step1PassProps> = ({ form, disabled })
                         label={formatMessage({ id: 'page.route.form.itemLabel.template' })}
                         name={field.name}
                         key={field.name}
-                        rules={[
```

Comment:
Just to confirm, this change doesn't require a change to the API interface?

Reply:
Not required.